### PR TITLE
Merging develop to master for v1.1.0

### PR DESCRIPTION
**Pull request by grikomsn**
Described on [v1.1.0 milestone](https://github.com/grikomsn/rotten-reviews/milestone/1)

### CODE REVIEW

**Comment by grikomsn** [comment]
We're closer to version 1.1.0! I'll try to ship the next version tomorrow Sunday, just need to update the readmes and review some PRs 😄

**Comment by grikomsn** [comment]
@fpereira1 shouldn't version 2 be shipped with there's a breaking change? We should try following https://semver.org

**Comment by fpereira1** [comment]
Yes. Good call for some reason I thought search would be a breaking change??

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (fpereira1) [inline .github/ISSUE_TEMPLATE/feature_request.md]: This doesn't seem necessary @grikomsn? it should be easy for people to submit an issue or new feature request. This templates may look to daunting and make people give up on giving their time to create an issue/feature

**Comment by vimark1** [comment]
@grikomsn Can we merge this PR?

**Comment by grikomsn** [comment]
Hello, everyone! Apologies for my absence, I got a lot of stuff to do and haven't touched this repo in a while. I'll review the issues and PRs today, we'll merge v1.1.0 this week. 🙏

**Pull request merged**